### PR TITLE
Update rolling branches

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

File: .github/workflows/autoupdate_submodules.yml [hunk 1]
```diff
@@ -14,14 +14,14 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - distro: foxy
-            branch: foxy
           - distro: humble
             branch: humble
+          - distro: iron
+            branch: iron
```
Comment (Acuadros95) [inline .github/workflows/autoupdate_submodules.yml:20]:
```suggestion
            branch: release/iron
```

File: .github/workflows/autoupdate_submodules.yml [hunk 2]
```diff
@@ -14,14 +14,14 @@ jobs:
...
           - distro: rolling
-            branch: main
+            branch: rolling
```
Comment (Acuadros95) [inline .github/workflows/autoupdate_submodules.yml:22]:
```suggestion
            branch: fix/rolling
```

**Review by Acuadros95 — COMMENTED** [review]
Testing purposes

File: .github/workflows/autoupdate_submodules.yml [hunk 3]
```diff
@@ -14,14 +14,14 @@ jobs:
...
         include:
-          - distro: foxy
-            branch: foxy
           - distro: humble
             branch: humble
+          - distro: iron
+            branch: release/iron
           - distro: rolling
-            branch: main
+            branch: fix/rolling
```
Comment (pablogs9) [inline .github/workflows/autoupdate_submodules.yml]: fix rolling?

**Review by pablogs9 — APPROVED** [review]

File: .github/workflows/autoupdate_submodules.yml [hunk 4]
```diff
@@ -14,14 +14,14 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - distro: foxy
-            branch: foxy
           - distro: humble
             branch: humble
+          - distro: iron
+            branch: release/iron
```
Comment (Acuadros95) [inline .github/workflows/autoupdate_submodules.yml]:
```suggestion
            branch: iron
```

File: .github/workflows/autoupdate_submodules.yml (hunk 3 above)
Comment (Acuadros95) [reply .github/workflows/autoupdate_submodules.yml]:
```suggestion
            branch: rolling
```

File: .github/workflows/build_agent.yml [hunk 5]
```diff
@@ -14,21 +14,18 @@ jobs:
...
       matrix:
         include:
-          - ros_distribution: foxy
-            branch: foxy
-            board_id: 20
           - ros_distribution: humble
             branch: humble
             board_id: 19
+          - ros_distribution: iron
+            branch: release/iron
```
Comment (Acuadros95) [inline .github/workflows/build_agent.yml]:
```suggestion
            branch: iron
```

File: .github/workflows/build_agent.yml [hunk 6]
```diff
@@ -14,21 +14,18 @@ jobs:
...
+            board_id: 20
           - ros_distribution: rolling
-            branch: main
+            branch: fix/rolling
```
Comment (Acuadros95) [inline .github/workflows/build_agent.yml]:
```suggestion
            branch: rolling
```

File: .github/workflows/ci.yml [hunk 7]
```diff
@@ -14,25 +14,22 @@ jobs:
...
       matrix:
         include:
-          - ros_distribution: foxy
-            branch: foxy
-            board_id: 20
           - ros_distribution: humble
             branch: humble
             board_id: 19
+          - ros_distribution: iron
+            branch: release/iron
```
Comment (Acuadros95) [inline .github/workflows/ci.yml]:
```suggestion
            branch: iron
```

File: .github/workflows/ci.yml [hunk 8]
```diff
@@ -14,25 +14,22 @@ jobs:
...
+            board_id: 20
           - ros_distribution: rolling
-            branch: main
+            branch: fix/rolling
```
Comment (Acuadros95) [inline .github/workflows/ci.yml]:
```suggestion
            branch: rolling
```

**Pull request merged**